tests/bench/obex.py

```
@@ -0,0 +1,643 @@
+    for r in results:
+        avg = (r.explicit_precision + r.explicit_recall) / 2
+        # + r.deductive_precision
+        # + r.deductive_recall
+        # / 4,
+        case_scores.append(avg)
+
+    output_data: dict[str, Any] = {
+        "metadata": {
+            "benchmark": "obex",
+            "description": "Observation Extraction Benchmark - evaluates deriver's ability to extract observations from messages",
+            "execution_timestamp": datetime.now().isoformat(),
```

Ruff (0.14.10)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)